.github/workflows/main.yml

```
@@ -32,5 +32,5 @@ jobs:
     strategy:
       matrix:
         node:
-          - lts/erbium
+          - lts/fermium
           - node
```

```suggestion
          - lts/fermium
```

This should silence the windows bug on Node 18

---

uh, sorry, the one with the `g`